src/MoonGuardTheme.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Taecontrol\MoonGuard;
+
+use Filament\Support\Assets\Theme;
+
+class MoonGuardTheme extends Theme
+{
+    public function getHref(): string
+    {
+        return asset($this->path);
+    }
+}
```

Comment:
What do we use this class for?

Reply:
to be able to dynamically load the theme